packages/core/src/utils/debugLogger.spec.ts

```diff
@@ -1,36 +1,36 @@
 import { debug, DebugLogger } from './debugLogger';
 import { describe, it, expect, vi } from 'vitest';
 
-describe('debugLogger', () => {
+describe.only('debugLogger', () => {
   it('should not log anything if debug is not enabled', () => {
     const consoleLogSpy = vi.spyOn(console, 'log');
     debug.log('test');
     expect(consoleLogSpy).not.toHaveBeenCalled();
   });
 
-  it('should log if debug is enabled', () => {
+  it.only('should log if debug is enabled', () => {
```

Comment:
```suggestion
  it('should log if debug is enabled', () => {
```
we should probably have an eslint rule for this :)

Reply:
Oops xD

Reply:
Made myself a memo for that